Doc/library/typing.rst

```diff
@@ -2013,6 +2013,12 @@ Introspection helpers
            'name': Annotated[str, 'some marker']
        }
 
+   .. note::
+
+      :func:`get_type_hints` does not work with imported :ref:`type aliases <type-aliases>` that include
+      forward references.
+      Consider avoiding forward references by enabling postponed evaluation of annotations (:pep:`563`) instead.
```

Comment:
Thanks for the docfix. One minor nit: 
```suggestion
      Enabling postponed evaluation of annotations (:pep:`563`) may remove the need for most forward references.
```
because [PEP 563 doesn't obviate](https://www.python.org/dev/peps/pep-0563/#forward-references) all uses of forward references.

Also please limit your lines to 80 characters, following the [style guide](https://devguide.python.org/documenting/#use-of-whitespace). Thank you!